_posts/2022-05-02-the-na-bug/the-na-bug.Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "Ooh na na... where are my sodium labs?"
+description: |
+  The NA bug, or, what happens when the same word is used in different contexts. [5 min read]
```

Comment:
I included the word count in the description.

In the future, we may want to look into updating the theme or distill functions to incorporate the reading time as an argument.

This is how it appears in the frontend:
![image](https://user-images.githubusercontent.com/4314491/166823026-af6218a1-feeb-441b-a2f0-dae120f79dee.png)